### PR TITLE
Story: Stop-sequence for python

### PR DESCRIPTION
**Pull request by santosh-kavhar**
Added support for stop sequence using python. We have made modification only in example_app.py. 
https://antaris.atlassian.net/browse/AT-4662

### CODE REVIEW

File: examples/app-python/example_app.py [hunk 1]
```diff
@@ -46,6 +51,39 @@ def handle_hello_friend(self, ctx):
         name = ctx.params
         logger.info(f"Handling sequence: hello, {name}!")
 
+    # Function to print numbers from 1 to 1000
```
Comment (rahulbhivare) [inline examples/app-python/example_app.py:54]: Change comment

File: examples/app-python/example_app.py [hunk 2]
```diff
@@ -208,6 +246,8 @@ def new():
     # Note : SatOS-Payload-SDK supports sequence upto 16 characters long
     app.mount_sequence("HelloWorld", ctl.handle_hello_world)
     app.mount_sequence("HelloFriend", ctl.handle_hello_friend)
+    app.mount_sequence("ThreadStart", ctl.handle_demo_thread_start)
```
Comment (deepaktawri) [inline examples/app-python/example_app.py:249]: Can we have some meaningful names for the sequences, like StartImaging, StopImaging etc ? Also we should get rid of examples of meaningless sequences like "HelloWorld", "HelloFriend" etc. Doesn't look enterprise ready example code.

Also, let's not do printing numbers kind of trivial work in handler functions. It's better to print a log msg and come out. We can put a detail comment there for users to stuff some meaningful code

**Review by deepaktawri — CHANGES_REQUESTED** [review]

File: examples/app-python/example_app.py (hunk 2 above)
Comment (santosh-kavhar) [reply examples/app-python/example_app.py:248]: This will be handled in separate ticket here: https://antaris.atlassian.net/browse/AT-5012

**Comment by santosh-kavhar** [comment]
Another approach is needed to handle the stop sequence logic

**Pull request closed without merging**